account_asset_management/models/account_asset.py

```diff
@@ -526,9 +526,22 @@ def group_lines(x, y):
             # recompute in case of deviation
             depreciated_value_posted = depreciated_value = 0.0
             if posted_lines:
+                total_table_lines = sum(
+                    [len(entry[1]["lines"]) for entry in enumerate(table)]
```

Comment:
Why using enumerate? It's not adding value and even complicating you. You don't need it for iterating.